src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * A record of ADD/DEL Persons
+ *
+ */
+public class Tagging {
+
+    private static final String ADD_NOTATION = "+";
+    private static final String DEL_NOTATION = "-";
+    private static final String BOX_PREFIX = "[";
+    private static final String BOX_SUFFIX = "]";
+    private static final String DELIMITER = " ";
+
+    private Person person;
+    private Tag tag;
+    private boolean isAdd_NOTATION;
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.